app/common/src/text/english.json

```diff
@@ -1,6 +1,10 @@
 {
   "submit": "Submit",
   "retry": "Retry",
+
+  "arbitraryFetchError": "An error occurred while fetching data",
+  "arbitraryFetchImageError": "An error occurred while fetching image",
```

Comment:
`image` -> `an image`

Reply:
Maybe, it should be `the` image? 🤔

Reply:
But I don't think it's a big deal here

Reply:
yeah this should probably be `the`. but yeah probably not a big deal